docs/reference/indices/rollover-index.asciidoc

```diff
@@ -233,14 +233,14 @@ PUT _index_template/template
   "template": {
     "mappings": {
       "properties": {
-        "@timestamp": {
+        "date": {
```

Comment:
I find the original name better, since it complies with ECS and also `date` opens up for a bit of confusion between field name and type.

Reply:
Agreed, but in the test data sets that is generated ([huge twitter setup](https://github.com/elastic/elasticsearch/blob/master/docs/build.gradle#L168)), has its timestamp in the date field. I think this should be changed in a followup change?